doc/releases/changelog-dev.md

```diff
@@ -166,65 +190,33 @@ such as `shots`, `rng` and `prng_key`.
+  [(#6707)](https://github.com/PennyLaneAI/pennylane/pull/6707)
 
-* `qml.equal` now supports `PauliWord` and `PauliSentence` instances.
-  [(#6703)](https://github.com/PennyLaneAI/pennylane/pull/6703)
+<h4>Transform for combining GlobalPhase instances</h4>
 
-* Remove redundant commutator computations from `qml.lie_closure`.
-  [(#6724)](https://github.com/PennyLaneAI/pennylane/pull/6724)
+* Added a new `qml.transforms.combine_global_phases` transform to combine all `qml.GlobalPhase` gates in a circuit into a single one applied at the end.
+  This can be useful for circuits that include a lot of `qml.GlobalPhase` gates, which can be introduced directly during circuit creation,
+  decompositions that include `qml.GlobalPhase` gates, etc.
+  [(#6686)](https://github.com/PennyLaneAI/pennylane/pull/6686)
 
-* Raises a comprehensive error when using `qml.fourier.qnode_spectrum` with standard numpy
-  arguments and `interface="auto"`.
-  [(#6622)](https://github.com/PennyLaneAI/pennylane/pull/6622)
+<h4>Better drawing functionality</h4>
 
 * Added support for the `wire_options` dictionary to customize wire line formatting in `qml.draw_mpl` circuit
   visualizations, allowing global and per-wire customization with options like `color`, `linestyle`, and `linewidth`.
   [(#6486)](https://github.com/PennyLaneAI/pennylane/pull/6486)
 
-* Added Pauli String representations for the gates X, Y, Z, S, T, SX, SWAP, ISWAP, ECR, SISWAP. Fixed a shape error in the matrix conversion of `PauliSentence`s with list or array input.
-  [(#6562)](https://github.com/PennyLaneAI/pennylane/pull/6562)
-  [(#6587)](https://github.com/PennyLaneAI/pennylane/pull/6587)
-  
-* `QNode` and `qml.execute` now forbid certain keyword arguments from being passed positionally.
-  [(#6610)](https://github.com/PennyLaneAI/pennylane/pull/6610)
-
-* Shortened the string representation for the `qml.S`, `qml.T`, and `qml.SX` operators.
-  [(#6542)](https://github.com/PennyLaneAI/pennylane/pull/6542)
-
-* Added JAX support for the differentiable Hartree-Fock workflow.
-  [(#6096)](https://github.com/PennyLaneAI/pennylane/pull/6096)
-  [(#6707)](https://github.com/PennyLaneAI/pennylane/pull/6707)
-
-* Added functions and dunder methods to add and multiply Resources objects in series and in parallel.
-  [(#6567)](https://github.com/PennyLaneAI/pennylane/pull/6567)
-
-* The `diagonalize_measurements` transform no longer raises an error for unknown observables. Instead,
-  they are left undiagonalized, with the expectation that observable validation will catch any undiagonalized
-  observables that are also unsupported by the device.
-  [(#6653)](https://github.com/PennyLaneAI/pennylane/pull/6653)
-
-* A `qml.wires.Wires` object can now be converted to a JAX array, if all wire labels are supported as 
-  JAX array elements.
-  [(#6699)](https://github.com/PennyLaneAI/pennylane/pull/6699)
-
 <h4>Capturing and representing hybrid programs</h4>
```

Comment:
Lots of stuff here! I would consider moving it up.

Reply:
This one we'll keep buried in the changelog because it's not meant to be that user-facing yet 😅. At some point we will need to make it more visible, though!